deluge/ui/baseargparser.py

```diff
@@ -20,6 +20,69 @@
 from deluge.log import setup_logger
 
 
+def find_subcommand(self, args=None):
+    """Find if a subcommand has been supplied.
+
+    Args:
+        args (list, optional): The argument list handed to parse_args().
+
+    Returns:
+        int: Index of the subcommand or '-1' if none found.
+
+    """
+    subcommand_found = -1
+    test_args = args if args else sys.argv[1:]
+
+    for x in self._subparsers._actions:
+        if not isinstance(x, argparse._SubParsersAction):
+            continue
+        for sp_name in x._name_parser_map.keys():
+            if sp_name in test_args:
+                subcommand_found = test_args.index(sp_name) + 1
+
+    return subcommand_found
+
+
+def set_default_subparser(self, name, args=None, before_subcmd=None):
+    """Sets the default argparse subparser.
+
+    Args:
+        name (str): The name of the default subparser.
+        args (list, optional): The argument list passed to parse_args().
+        before_subcmd (list): The arguments to test for that come before the subcommand.
+
+    Returns:
+        bool: True if help option is not found after a subcommand, else False.
```

Comment:
Is this text about the help option correct. It doesn't seem to have any relevance here?